packages/atomic/src/components/facets-v1/atomic-rating-facet/atomic-rating-facet.tsx

```diff
@@ -69,6 +69,32 @@ export class AtomicRatingFacet
   @State() public error!: Error;
   @State() public isCollapsed = false;
 
+  private iconActive = (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="12"
+      height="13"
+      viewBox="0 0 12 13"
+      fill="none"
+      class="star-active"
+    >
+      <path d="M2.12261 7.32528L0.386144 5.64869C-0.196726 5.08592 0.0965754 4.09991 0.892272 3.94721L3.52262 3.44244C3.84159 3.38123 4.11096 3.16892 4.24501 2.87309L5.08915 1.01016C5.44396 0.227133 6.55604 0.227132 6.91085 1.01016L7.75499 2.87309C7.88904 3.16892 8.15841 3.38123 8.47738 3.44244L11.1077 3.94721C11.9034 4.09991 12.1967 5.08592 11.6139 5.64869L9.87739 7.32528C9.64107 7.55345 9.53188 7.88314 9.58529 8.20726L10.0399 10.966C10.1756 11.79 9.3013 12.4076 8.57008 12.0042L6.48308 10.8527C6.18239 10.6868 5.81761 10.6868 5.51692 10.8527L3.42992 12.0042C2.6987 12.4076 1.82437 11.79 1.96014 10.966L2.41471 8.20726C2.46812 7.88314 2.35893 7.55345 2.12261 7.32528Z" />
+    </svg>
+  );
+  private iconInactive = (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      width="12"
+      height="13"
+      viewBox="0 0 12 13"
+      fill="none"
+      class="star-inactive"
+    >
+      <path d="M2.12261 7.32528L0.386144 5.64869C-0.196726 5.08592 0.0965754 4.09991 0.892272 3.94721L3.52262 3.44244C3.84159 3.38123 4.11096 3.16892 4.24501 2.87309L5.08915 1.01016C5.44396 0.227133 6.55604 0.227132 6.91085 1.01016L7.75499 2.87309C7.88904 3.16892 8.15841 3.38123 8.47738 3.44244L11.1077 3.94721C11.9034 4.09991 12.1967 5.08592 11.6139 5.64869L9.87739 7.32528C9.64107 7.55345 9.53188 7.88314 9.58529 8.20726L10.0399 10.966C10.1756 11.79 9.3013 12.4076 8.57008 12.0042L6.48308 10.8527C6.18239 10.6868 5.81761 10.6868 5.51692 10.8527L3.42992 12.0042C2.6987 12.4076 1.82437 11.79 1.96014 10.966L2.41471 8.20726C2.46812 7.88314 2.35893 7.55345 2.12261 7.32528Z" />
```

Comment:
If the class attribute is the only thing that is different, we could also create a function. e.g.

```
function buildStar(className: string) {
   return <svg ...></svg>
}

const activeStar = buildStar('star-active');
const inactiveStar = ...
```

Reply:
If I am able to implement this with the partially filled stars as well, I will implement it. Otherwise, I think it would be weird in terms of code structure that some stars can be built using a function while the partially filled ones cannot.

Reply:
It may be easier not to have separate icons for partially filled stars and instead clip and overlay them with CSS.

Reply:
For a fully filled an an empty star, the only difference is the fill so this can be done with only CSS but for a partially filled star, it requires an additional HTML element for the linear gradient so I'm not sure how we could do that with only CSS.